spec/functions/find_or_create_password_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
+require 'tmpdir'
+
+describe 'find_or_create_password' do
```

Comment:
This could probably be removed instead of adding tests for, we no longer use it anywhere that I am aware of.